crypto/fipsmodule/ec/make_tables.go

```diff
@@ -200,27 +200,35 @@ func writeP384Table(path string) error {
 
 // This file is generated by make_tables.go.
 
-// Base point pre computation
-// --------------------------
+// P-384 base point pre computation
+// --------------------------------
 //
-// The pre-computed table for the base point G of P-384, fiat_p384_g_pre_comp,
-// consists of 20 sub-tables, each holding 16 points. A point is represented
+// The precomputed table for the base point G of P-384, fiat_p384_g_pre_comp,
+// consists of 14 sub-tables, each holding 64 points. A point is represented
 // by a pair of field elements (x, y).
 //
 // The j-th point of the i-th sub-table is:
 //     fiat_p384_g_pre_comp[i][j] = [(2j + 1)2^{20i}]G.
-// The table is populated with such points for i in [0, 19] and j in [0, 15];
-// and used in mul_base and mul_public functions in p384.c for computing
+// The table is populated with such points for i in [0, 13] and j in [0, 63];
+// and used in mul_base and mul_public functions in |p384.c| for computing
 // a scalar product with the Comb method (see the functions for details).
+//
+// The table and the method for computing the scalar products is taken from
```

Comment:
I would say here "The table and its usage in scalar multiplications are adapted from ..."

Reply:
done